fe/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -715,7 +714,7 @@ public static void checkAndCreateSource(Database db, DataDescription dataDescrip
             // check hll
```

Comment:
`hll_hash` is a kind of hadoop function, so here we use `columnToHadoopFunction` to check only HLL column is correct.

Reply:
Yep